public/app/features/alerting/unified/components/rule-editor/LabelsField.tsx

```diff
@@ -271,71 +261,75 @@ const LabelsField: FC<Props> = ({ dataSourceName }) => {
           </Text>
-          <Tooltip
-            content={
-              <div>
-                The dropdown only displays labels that you have previously used for alerts. Select a label from the
-                dropdown or type in a new one.
-              </div>
-            }
-          >
-            <Icon className={styles.icon} name="info-circle" size="sm" />
-          </Tooltip>
+          <NeedHelpInfo
+            contentText="The dropdown only displays labels that you have previously used for alerts. 
+            Select a label from thedropdown or type in a new one."
+            title="Labels"
+          />
         </Stack>
-      </Label>
+      </div>
+      <div className={styles.labelsContainer}></div>
       {dataSourceName ? <LabelsWithSuggestions dataSourceName={dataSourceName} /> : <LabelsWithoutSuggestions />}
     </div>
   );
 };
 
 const getStyles = (theme: GrafanaTheme2) => {
   return {
-    icon: css`
-      margin-right: ${theme.spacing(0.5)};
-    `,
-    flexColumn: css`
-      display: flex;
-      flex-direction: column;
-    `,
-    flexRow: css`
-      display: flex;
-      flex-direction: row;
-      justify-content: flex-start;
-
-      & + button {
-        margin-left: ${theme.spacing(0.5)};
-      }
-    `,
-    deleteLabelButton: css`
-      margin-left: ${theme.spacing(0.5)};
-      align-self: flex-start;
-    `,
-    addLabelButton: css`
-      flex-grow: 0;
-      align-self: flex-start;
-    `,
-    centerAlignRow: css`
-      align-items: baseline;
-    `,
-    equalSign: css`
-      align-self: flex-start;
-      width: 28px;
-      justify-content: center;
-      margin-left: ${theme.spacing(0.5)};
-    `,
-    labelInput: css`
-      width: 175px;
-      margin-bottom: -${theme.spacing(1)};
-
-      & + & {
-        margin-left: ${theme.spacing(1)};
-      }
-    `,
+    icon: css({
+      marginRight: theme.spacing(0.5),
+    }),
+    flexColumn: css({
+      display: 'flex',
+      flexDirection: 'column',
+    }),
+    flexRow: css({
+      display: 'flex',
+      flexDirection: 'row',
+      justifyContent: 'flex-start',
+      '& + button': {
+        marginLeft: theme.spacing(0.5),
+      },
+    }),
+    deleteLabelButton: css({
+      marginLeft: theme.spacing(0.5),
+      alignSelf: 'flex-start',
+    }),
+    addLabelButton: css({
+      flexGrow: 0,
+      alignSelf: 'flex-start',
+    }),
+    centerAlignRow: css({
+      alignItems: 'baseline',
+    }),
+    equalSign: css({
+      alignSelf: 'flex-start',
+      width: '28px',
+      justifyContent: 'center',
+      marginLeft: theme.spacing(0.5),
+    }),
+    labelInput: css({
+      width: '175px',
+      marginBottom: `-${theme.spacing(1)}`,
+      '& + &': {
+        marginLeft: theme.spacing(1),
+      },
+    }),
+    labelsContainer: css({
+      marginBottom: theme.spacing(3),
+    }),
+    labelsDescription: css({
```

Comment:
nit: Can we use the `Stack` component instead of adding more styles? I don't see anything here that would require custom styling :)